Corrfunc/mocks/DDtheta_mocks.py

```diff
@@ -351,6 +353,278 @@ def DDtheta_mocks(autocorr, nthreads, binfile,
         return results, api_time
 
 
+def find_fastest_DDtheta_mocks_bin_refs(autocorr, nthreads, binfile,
+                                        RA, DEC,
+                                        verbose=False, output_thetaavg=False,
+                                        max_cells_per_dim=100,
+                                        isa=r'fastest',
+                                        maxbinref=3, nrepeats=3,
+                                        return_runtimes=False):
+    """
+
+    Parameters
+    ----------
+    autocorr : boolean, required
+        Boolean flag for auto/cross-correlation. If autocorr is set to 1,
+        then the second set of particle positions are not required.
+
+    nthreads : integer
+        Number of threads to use.
+
+    binfile: string or an list/array of floats. Units: degrees.
+        For string input: filename specifying the ``theta`` bins for
+        ``DDtheta_mocks``. The file should contain white-space separated values
+        of (thetamin, thetamax)  for each ``theta`` wanted. The bins need to be
+        contiguous and sorted in increasing order (smallest bins come first).
+
+        For array-like input: A sequence of ``theta`` values that provides the
+        bin-edges. For example,
+        ``np.logspace(np.log10(0.1), np.log10(10.0), 15)`` is a valid
+        input specifying **14** (logarithmic) bins between 0.1 and 10.0
+        degrees. This array does not need to be sorted.
+
+    RA : array-like, real (float/double)
```

Comment:
```suggestion
    RA : array-like, real (float/double)
```
This should probably be in `RA1`. Similarly for "DEC" -> `DEC1`

Reply:
Yes, I have made the naming changes accordingly in the latest batch of (squashed) commits.